dev/benchmarks/macrobenchmarks/lib/main.dart

```diff
@@ -137,6 +137,13 @@ class HomePage extends StatelessWidget {
               Navigator.pushNamed(context, kFadingChildAnimationRouteName);
             },
           ),
+          RaisedButton(
```

Comment:
This change is already in the tree. If you remove these lines and rebase on the latest Top of Tree it should be there.